### ISSUE TITLE
ArduinoJson not being listed

### ISSUE THREAD

**Issue by bblanchon** [msgpack/msgpack#274]
Hi!

I followed [these instructions](https://github.com/msgpack/website/blob/master/README.md#how-to-list-up-your-project-on-msgpackorg) to get [ArduinoJson](https://github.com/bblanchon/ArduinoJson) listed on msgpack.org, but it didn't work.
Did I miss something?

BTW, I wish we could get listed on msgpack.org without changing the description of the repo because it feels very awkward for ArduinoJson. Maybe we could use GitHub topics, for example.

Best Regards,
Benoit

**Comment by repeatedly** [comment]
It seems a crawler side issue.

@frsyuki Could you check the crawler?

**Comment by frsyuki** [comment]
Yes. [Crawler](https://github.com/msgpack/website/blob/master/update-index.rb) may have a problem. Let me check.
I hope it doesn't need a overhaul...

**Comment by kriszyp** [comment]
I think this is the same as #289. @repeatedly and @frsyuki I know you are both probably very busy, but it would be really great to be able to get the list of implementations updated. There is a lot of out of date stuff on there, and lot of better implementations now. Thank you!

**Comment by dRomke** [comment]
Bump!
Would be nice to have the list of implementations back up on the site.
Seems like at some point in januari 2021 it was working fine: https://web.archive.org/web/20210107193417/https://msgpack.org/
